tests/conftest.py

```diff
@@ -213,56 +215,85 @@ def exposure_filter_array(exposure_factory):
     yield e
 
 
-@pytest.fixture
-def decam_example_file():
+def get_decam_example_file():
     filename = os.path.join(CODE_ROOT, 'data/DECam_examples/c4d_221104_074232_ori.fits.fz')
     if not os.path.isfile(filename):
-        url = 'https://astroarchive.noirlab.edu/api/retrieve/004d537b1347daa12f8361f5d69bc09b/'
-        response = wget.download(
-            url=url,
-            out=os.path.join(CODE_ROOT, 'data/DECam_examples/c4d_221104_074232_ori.fits.fz')
-        )
-        assert response == filename
-
-    yield filename
-
+        cachedfilename = f'{filename}_cached'
+        if not os.path.isfile( cachedfilename ):
+            url = 'https://astroarchive.noirlab.edu/api/retrieve/004d537b1347daa12f8361f5d69bc09b/'
+            response = wget.download( url=url, out=cachedfilename )
+            assert response == cachedfilename
+        os.symlink( cachedfilename, filename )
+    return filename
```

Comment:
For speed in tests.  The original one was getting deleted with one of the datastore deletion/cleanup calls, since we were downloading it straight to where it got loaded into the database.